tools/cross/format.py

```diff
@@ -97,9 +97,13 @@ def clang_format(files: List[str], check: bool = False) -> bool:
 def prettier(files: List[str], check: bool = False) -> bool:
     cmd = [PRETTIER]
     if check:
-        cmd.append("--check")
+        # This flag will only print files that require formatting
+        # and change exit code of the process depending on state.
+        cmd.append("--list-different")
```

Comment:
```suggestion
        cmd.append("--check")
```

I'd prefer the output of check because it gives you some context as to what it's printing.